src/libstd/primitive_docs.rs

```diff
@@ -10,8 +10,46 @@
 
 #[doc(primitive = "bool")]
 //
-/// The boolean type.
+/// The boolean type represents a value, which could only be either `true` or `false`.
 ///
+/// It implements the various traits, such as [`BitAnd`], [`BitOr`], [`Not`], etc., to deal with
+/// the boolean operations. [`if`] always demands a `bool` value.
+///
+/// [`if` conditionals]: ../../book/if.html
+/// [`BitAnd`]: ../ops/trait.BitAnd.html
+/// [`BitOr`]: ../ops/trait.BitOr.html
+/// [`Not`]: ../ops/trait.Not.html
+///
+/// # Examples
+///
```

Comment:
It might be nice to have a "Basic usage:" example which just shows assigning true or false, and comparing them. I dunno. It's tough. Thoughts? I'm on the fence.